tests/functionality_tests/pid_tuning.py

```diff
@@ -1,20 +1,31 @@
 import math
 
+from electrical.motor_controller import MotorController
 from engine.grid import Grid
 from engine.database import DataBase
 from engine.robot import Robot, Phase
 
 import matplotlib.pyplot as plt
 import matplotlib.animation as animation
 from matplotlib import style
+import time
+
+from electrical.gps import GPS
+from electrical.imu import IMU
+
+import serial
+import busio
+import board
+
+import csv
```

Comment:
not sure where serial, busio, and board is used. I think they were used when using the other motorcontroller class. looks like there's an extra import csv